pkg/server/datastore.go

```diff
@@ -201,18 +225,26 @@ func (s *Server) DeleteDataStore(ctx context.Context, req *sdcpb.DeleteDataStore
 
 	err = ds.Stop()
 	if err != nil {
-		log.Errorf("failed to stop datastore %s: %v", name, err)
+		//TODO: Should we return error here and stop execution?
```

Comment:
My impression was no... since the deletion failed but the intention is to be deleted ... we should at least remove it form the datastores listed in data-server... agree?